cpu/esp32/doc.txt

```diff
@@ -1256,6 +1259,43 @@ In this case the ESP-NOW interface must use the same channel as the AP of the
 infrastructure WiFi network. All ESP-NOW nodes must therefore be compiled with
 the channel of the AP as value for the parameter 'ESP_NOW_CHANNEL'.
 
+\anchor esp32_wifi_ap_network_interface
+## <a name="esp32_wifi_ap_network_interface"> WiFi SoftAP Network Interface </a> &nbsp;[[TOC](#esp32_toc)]
+
+The RIOT port for the ESP32 supports a `netdev` interface for the ESP32 WiFi
+SoftAP mode. Module `esp_wifi_ap` has to be enabled to use it.
+
+The following parameters can be configured:
+
+<center>
+
+| Parameter          | Default          | Description
+:--------------------:------------------:-------------
+ESP_WIFI_SSID        | "RIOT_AP"        | Static SSID definition for the SoftAP
+ESP_WIFI_AP_PREFIX   | "RIOT_AP_"       | Optional prefix for dynamic SSID, if used, the node will create the SSID based on the prefix + mac address (e.g.: "RIOT_AP_aabbccddeeff"). This is disabled by default and ```ESP_WIFI_SSID``` is used, define this to enable the usage of the SSID prefix.
+ESP_WIFI_PASS        | none             | The password for the WiFi SoftAP network interface. If no password provided the interface will be "open", otherwise it uses WPA2-PSK authentication mode.
+ESP_WIFI_SSID_HIDDEN | not hidden       | Whether the SoftAP SSID should be hidden.
```

Comment:
?
```suggestion
ESP_WIFI_SSID_HIDDEN | 0                | Whether the SoftAP SSID should be hidden.
```

Reply:
Seems right, I've put `not hidden` because I didn't know if it was good to put to `0` on it or that it could result confusing.

Reply:
I guess someone would understand that a 0 means false and another value means true. Alternatively, you could say:
```
ESP_WIFI_SSID_HIDDEN | 0                | Whether the SoftAP SSID should be hidden (0 - not hidden, 1 - hidden).
```